src/main/java/org/csekuaa/backend/model/Alumni.java

```diff
@@ -5,106 +5,97 @@
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
+import java.sql.Date;
 import java.sql.Timestamp;
-import java.time.LocalDate;
-import java.time.LocalDateTime;
+import java.util.Set;
 
 @NoArgsConstructor
 @Getter
 @Setter
-@Table(name = "alumni")
 @Entity
+@Table(name = "alumni")
 public class Alumni {
 
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
     @Id
-    @Column(name = "alumni_id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "alumni_id", nullable = false)
     private int alumniId;
-
-    @Column(name = "roll")
+     
+    @Column(name = "roll", nullable = false, length = 10)
     private String roll;
-
-    @Column(name = "name")
+     
+    @Column(name = "name", nullable = false, length = 100)
     private String name;
```

Comment:
full_name should be better.

Reply:
ok i will consider it as fullName in Entity,